### DISCUSSION TITLE
Choose the next/previous page

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [adityatelange/hugo-PaperMod#629] [Ideas]
I am aware that there exists a weight parameter that would technically allow for this but there are cases where that might not work, for example: you might want some series of posts to appear ordered from first to last in your front page (which works without changing the weights) but if you click any of them, the "PREV PAGE" and "NEXT PAGE" buttons at the bottom appear inverted relative to the ordering of the pots

Example:
posts:
- `PostFromMySeries2`
- `PostFromMySeries1`

the `PostFromMySeries1` page will have the "PREV PAGE" button point to `PostFromMySeries2`

Solution: Add a parameter for overriding these 2 buttons with whatever post you want

(if there is currently a way to do this as of now that I am not aware of, let me know!)

**Comment by danielfdickinson** [comment]
Yeah, I find Hugo's definition of previous and next rather confusing too. I just switch them around (but label them as they are now) so Hugo's Prev is used for the button with the text 'Next', and vice versa).

**Comment by adityatelange** [comment]
> Solution: Add a parameter for overriding these 2 buttons with whatever post you want

I don't think that is easy and will have an effect on site generation as we would have to parse each and every page to select page what one wants to display there, and I don't see that is an easy solution after all.

I'll see if we can still add it in the future if hugo has some builtin implementations for this
